app/assets/javascripts/discourse/app/components/groups-form-interaction-fields.js

```diff
@@ -98,6 +98,11 @@ export default Component.extend({
     return (!isCreated && canCreate) || (isCreated && canAdmin);
   },
 
+  @discourseComputed("model.can_edit_group")
+  canEditGroup(canEdit) {
```

Comment:
Thanks, @nbianca! First time working with Ember, and today I learned about aliases. 🙂

Reply:
I would recommend against aliases. prefer this instead: https://api.emberjs.com/ember/release/functions/@ember%2Fobject%2Fcomputed/reads

Problem with `alias` is that it's bi-directional, meaning that if you alias `foo.bar.baz` to `cat`, setting `cat` will also change `foo.bar.baz`, first you probably don't need this behavior and then it's a very risky behavior as it can mutate things very far away from where the actual mutation happens. Generally Ember prefers to follow `DDAU` (data down action up pattern) where you use actions to bubble up where the variable has been created and mutate here, instead of mutating it very far down the chain.

Let me know if it's not clear enough.

Reply:
Ah. Yes. I'm familiar with two-way bindings (and its caveats) from Angular. 🙂